utils/lxc/files/lxc-auto.init

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh /etc/rc.common
 
-. /lib/functions.sh
+. "${IPKG_INSTROOT}"/lib/functions.sh
```

Comment:
Are you sure that this https://github.com/openwrt/packages/blob/master/utils/lxc/Makefile#L65 does not help for you?  That's why I asked for your commit hash of the packages feed, which you did not provide me neither the LXC Makefile.

Because it was fixed by https://github.com/openwrt/packages/commit/039912dec5d3ba2b0f6f53ab8330ab9fea2f7adf

Reply:
That is a very different failure: postinstall does NOT run by image builder.

I did provide a hash of OpenWrt to show that it is the latest with the latest of packages. Either way:
```
OPENWRT_RELEASE="OpenWrt 22.03-SNAPSHOT r19691-39c115028a"
packages: commit 85e6f9348a3df7ec7d8136476b688b56fea7ad0a (HEAD -> openwrt-22.03, origin/openwrt-22.03)
```
I just ran a image builded build again and got this:

```
Finalizing root filesystem...
/build/openwrt-imagebuilder/build_dir/target-x86_64_musl/root-x86/etc/init.d/lxc-auto: line 3: /lib/functions.sh: No such file or directory
Enabling boot
Enabling cgroupfs-mount
Enabling collectd
Enabling cron
Enabling ddns
Enabling dnsmasq
Enabling dockerd
Enabling done
Enabling dropbear
Enabling firewall
Enabling fstab
Enabling gpio_switch
Enabling led
Enabling lm-sensors
Enabling log
Enabling luci_statistics
./etc/init.d/lxc-auto: line 3: /lib/functions.sh: No such file or directory
Enabling lxc-auto
```

Reply:
BTW, the comment in the commit you referenced explicitly states that the image builder build is not fixed.

```
Sourcing /lib/functions.sh is not needed, as /etc/rc.common does so
already. Unfortunately removing that line from the init script is not
enough to fix the problem. The postinst script should also check
IPKG_INSTROOT. As these two changes are unrelated, they should go in
separate commits, and the solution to the image build problem is to
revert the commit that introduced the breakage.```

Reply:
Try this:

```
wget https://downloads.openwrt.org/releases/22.03-SNAPSHOT/targets/x86/64/openwrt-imagebuilder-22.03-SNAPSHOT-x86-64.Linux-x86_64.tar.xz
tar zxf openwrt-imagebuilder-22.03-SNAPSHOT-x86-64.Linux-x86_64.tar.xz
cd openwrt-imagebuilder-22.03-SNAPSHOT-x86-64.Linux-x86_64
make image PROFILE=generic BIN_DIR=./firmware FILES=./files PACKAGES="lxc-auto"
```

Reply:
BTW, the same issue will be happening if you just build a firmware with `lxc-auto` built in.

Reply:
Here: https://openwrt.org/docs/guide-developer/procd-init-scripts#init_scripts_during_compilation

Reply:
@BKPepe, are there still open questions?

Reply:
@BKPepe , do you still have concerns about this change?